import-export-cli/cmd/k8s/k8s.go

```diff
@@ -19,21 +19,27 @@
 package k8s
 
 import (
+	"bytes"
 	"github.com/spf13/cobra"
+	k8sUtils "github.com/wso2/product-apim-tooling/import-export-cli/operator/utils"
 	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"io"
+	"os"
+	"os/exec"
 )
 
 // K8s command related usage Info
 const K8sCmdLiteral = "k8s"
 const k8sCmdShortDesc = "Kubernetes mode based commands"
 
-const k8sCmdLongDesc = `Kubernetes mode based commands such as install, uninstall, add/update api, change registry.`
+const k8sCmdLongDesc = `Kubernetes mode based commands such as add, update and delete API`
 
-const k8sCmdExamples = utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sInstallCmdLiteral + ` ` + K8sInstallApiOperatorCmdLiteral + `
-` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sUninstallCmdLiteral + ` ` + K8sUninstallApiOperatorCmdLiteral + `
-` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sAddCmdLiteral + ` ` + AddApiCmdLiteral + ` ` + `-n petstore --from-file=./Swagger.json --replicas=1 --namespace=wso2
-` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sUpdateCmdLiteral + ` ` + AddApiCmdLiteral + ` ` + `-n petstore --from-file=./Swagger.json --replicas=1 --namespace=wso2
-` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sChangeCmdLiteral + ` ` + K8sChangeDockerRegistryCmdLiteral
+const k8sCmdExamples = utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sAddCmdLiteral + ` ` + AddApiCmdLiteral + ` ` +
+	`-n petstore --file=./Swagger.json --namespace=wso2
+` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + K8sUpdateCmdLiteral + ` ` + AddApiCmdLiteral + ` ` +
+	`-n petstore --file=./Swagger.json --namespace=wso2
+` + utils.ProjectName + ` ` + K8sCmdLiteral + ` ` + k8sDeleteCmdLiteral + ` ` + k8sDeleteAPICmdLiteral + ` ` +
+	`-n petstore`
```

Comment:
Let's make these examples consistent as well? (Short flags and long flags separation)

Reply:
Fixed with aa03cd8b4d47fd7fec7dc79a709f5a9f8cba9fd1